File: internal/sast/fixtures/python/xss_positive.py
# internal/sast/fixtures/python/xss_positive.py
from flask import Flask, request, render_template_string, Markup

app = Flask(__name__)

@app.route("/echo")
def echo():
    msg = Markup(request.args.get("msg", ""))
    return render_template_string("<p>{{ msg }}</p>", msg=msg)

File: internal/sast/fixtures/csharp/Xss_positive.cs
// internal/sast/fixtures/csharp/Xss_positive.cs
using Microsoft.AspNetCore.Mvc;
using Microsoft.AspNetCore.Html;

public class XssController : Controller
{
    public IActionResult Echo(string msg)
    {
        ViewBag.Msg = new HtmlString(msg);
        return View();
    }
}
